.eslintrc.js

```diff
@@ -29,5 +29,6 @@ module.exports = {
     "import/no-cycle": "error",
     "no-unused-vars": "off",
     "prefer-template": "error",
+    "crlf": "auto",
```

Comment:
Can't find the eslint rule for this. What will happen if this rule isn't here?

Reply:
Removed for now, will see how to fix it on a git level on my side.